fedbiomed/researcher/cli.py

```diff
@@ -54,9 +58,11 @@ def start(self, args):
         if args.directory:
```

Comment:
above, it says that default directory is the notebook directory, but in my tests it is the fedbiomed root directory but the root directory (could not extend test because atm `--path` is not working)

Reply:
Yes, the help text should be updated. It is the directory where the command is executed.